src/modules/services/minio.nix

```diff
@@ -72,6 +76,14 @@ in
       type = types.package;
       description = lib.mdDoc "Minio package to use.";
     };
+
+    ensureBuckets = lib.mkOption {
```

Comment:
We could call it just `buckets` as all devenv settings are about ensuring the state :)

Reply:
done :)